qbraid_qir/exceptions.py

```diff
@@ -24,3 +24,7 @@ class QbraidQirError(Exception):
 
 class QirConversionError(QbraidQirError):
     """Class for errors raised when converting quantum program to QIR."""
+
+
+class InvalidSquinInput(QbraidQirError):
+    """Class for errors raised when invalid input is provided to squin converter."""
```

Comment:
This custom exception should go in squin module e.g. `qbraid_qir/squin/exceptions.py`